test/src/parsing.py

```diff
@@ -84,6 +84,7 @@ def test_newline_after_nul(self):
         resp = ('MESSAGE\n'
                 'destination:/exchange/amq.fanout\n'
                 'message-id:Q_/exchange/amq.fanout@@session-(.*)\n'
+                'redelivered:false\n'
```

Comment:
It's just some tests assert on the exact frame payload. Clients won't break because the header is optional.

> On 16/9/2015, at 20:59, Alvaro Videla <notifications@github.com> wrote:
> 
> In test/src/parsing.py:
> 
> > @@ -84,6 +84,7 @@ def test_newline_after_nul(self):
> >          resp = ('MESSAGE\n'
> >                  'destination:/exchange/amq.fanout\n'
> >                  'message-id:Q_/exchange/amq.fanout@@session-(.*)\n'
> > +                'redelivered:false\n'
> if we have to change all the tests do account for redelivered, does this means this will break things to users? If yes, should this go to master instead?
> 
> —
> Reply to this email directly or view it on GitHub.
>